tests/subsys/dfu/mcuboot/src/main.c

```diff
@@ -117,7 +118,12 @@ void test_write_confirm(void)
 		zassert_true(ret == 0, "Write to flash");
 	}
 
-	zassert(boot_write_img_confirmed() == 0, "pass", "fail");
+	/* set copy-done flag */
+	ret = flash_area_write(fa, fa->fa_size - 32, &flag, sizeof(flag));
```

Comment:
This is a bit confusing to me, you're writing to the correct location but you're writing 4 bytes (`uint32_t`) instead of the usual 8 bytes that MCUboot uses for a flag, and even if it works (with the 0x01 at the right place) it will only work for one specific endianness.

Reply:
I believe this test-sute entirely written like that (assues 32 bits write access and lite-endianes).
I can rework it to be more widely, but see this as another PR.
Is that Ok for you?

Reply:
It is acceptable, but I don't see why not just fixing it properly since it's two lines of change.

Reply:
ok. done